tests/integration/loopback_test.yaml

```diff
@@ -0,0 +1,34 @@
+description: Revert the loopback interface to default
```

Comment:
There should be a comment at the beginning that explains the purpose of this file.

Reply:
I added the comment at the beginning that explains the purpose of this file.